<html>
<header><title>Model</title></header>
<body>
<center>
<img src="images/logo.png">
<br>
<h2>Introduction</h2>
<hr>
</center>
<p>
				SeqVis, a Java standalone application, is an interactive three-dimensional visualization tool
				to explore compositional heterogeneity in large alignment of nucleotide sequences. 
				Existing methods for assessing compositional heterogeneity among nucleotide sequences are either 
				not reliable or computationally expensive for large alignments. SeqVis visualizes nucleotide compositions
				in a tetrahedron model (extension of the de-Finetti plot). The user-friendly 
				features provided by SeqVis allows compositional heterogeneous sequences to be 
				visually identified. 
				</p>
				
				<p>
				<center>
					<img src="images/main2.png"><br>
					<font size="2">
					<b>Fig. 1. Snapshot of SeqVis. </b> Visualizing compositional heterogeneity 
					of different species across the three codon sites.
					</font>
				</center>
				</p>
				
				
				<h2>Background</h2>
				<h3 id="compHetero">Compositional heterogeneity</h2>
				<p>
				Most phylogenetic methods assume that the sequences
				evolved under a single time-reversible Markov process (homogeneous, stationary, and reversible conditions).
				Compositional heterogeneity (ie, significant deviation of frequency of nucleotide A,T,G and C) 
				in sequence data suggest that they did not evolve under these conditions and therefore phylogeny 
				may not be accurately inferred. 
				</p>
				
				<h3 id="methods">Existing methods for assessing compositional heterogeneity</h2>
				<p>
				Currently, there are four categories of methods that detect compositional heterogeneity 
				(Jermiin <i>et al</i>., 2004)in the alignments of 
				nucleotides. The first category uses graphs or tables to visualize the compositional heterogeneity. 
				Other categories perform evaluations against the expected distributions based on test statistics. 
				The use of the first category is fairly limited to some species, whereas the latter is 
				usually either statistically invalid or not accommodated by the scientific community. 
				Matched-pairs tests of homogeneity for analyzing aligned nucleotides were inspired from the second problem. 
				These tests provide useful details on Markov process. However, the results of data containing 
				many sequences may be impractical.
				</p>
				
				<hr>
				
				</p>
			  <p>
			  Jermiin,L.S. et al. (2004). The biasing effect of compositional heterogeneity on
			  			   phylogenetic estimates may be underestimated. Syst. Biol., 53, 638-644.
		   	  </p>
</body>
</html>